django/forms/fields.py

```diff
@@ -343,7 +343,7 @@ def to_python(self, value):
             return None
         if self.localize:
             value = formats.sanitize_separators(value)
-        value = str(value).strip()
+        value = str(value)
```

Comment:
Thanks :+1: IMO, we can also remove this assignment and simplify the code:
```python
        ...
        if self.localize: 
             value = formats.sanitize_separators(value)
        try: 
             return Decimal(str(value)) 
        except DecimalException:
            raise ValidationError(self.error_messages['invalid'], code='invalid')
```

Reply:
Just came here to suggest the exact same thing!